app.py

```diff
@@ -106,6 +116,10 @@ def create_journal_entry(user_id, plant_id):
     user = mongo.db.users.find_one_or_404({'_id': user_id})
     plant = mongo.db.plants.find_one_or_404({'_id': plant_id})
     timestamp = int(datetime.now().timestamp() * 1000)
+
+    if request.json.get('entryType') == 'image':
+        mongo.db.plants.update_one({'_id': plant_id}, {'$set': {'image_url': request.json.get('info')}})
+
```

Comment:
I think this is also not going to work with our current schema, info is a dict with properties of 'imageURL' and 'notes' we can look into un-nesting these if it is going to be a problem

Reply:
No need to look at un-nesting it. I changed it to account for nesting.